source/software/python_package_management.rst

```diff
@@ -184,42 +184,71 @@ site <https://conda.readthedocs.io/en/latest/>`_.
 Alternatives to conda
 ---------------------
 
-Setting up your own package repository for Python is straightforward.
+Setting up your own package repository for Python is straightforward. 
+`PyPi, the Python Package Index <https://pypi.org/>`_ is a web repository of
+Python packages and you can easily install packages from it using either
+``easy_install`` or ``pip``. In both cases, you'll have to create a 
+subdirectory for Python in your ``${VSC_DATA}`` or ``${VSC_HOME}`` directory 
+(not all sites allow to install much in ``${VSC_HOME}``), add this directory
```

Comment:
I'd suggest to eliminate the reference to $VSC_HOME altogether.  The Python directory easily grows too large anyway.

Reply:
I've added a new commit that does just that for the "Alternatives to Conda" section. Someone at KU Leuven should check the last section, Installing Anaconda on NX node (KU Leuven Thinking), though.